exercise1.js

```diff
@@ -0,0 +1,5 @@
+function calculateAverage(numbers) {
+    const sum = numbers.reduce((add, number) => add + number, 0);
+    const average = sum / numbers.length;
+    return average;
```

Comment:
It may be better to remove these variables like this, 
```suggestion
    return numbers.reduce((add, number) => add + number, 0) / numbers.length;
```